Documentation/development-flow.md

```diff
@@ -33,11 +33,41 @@ cd $GOPATH/src/github.com/rook
 
 # Clone your fork, where <user> is your GitHub account name
 git clone https://github.com/<user>/rook.git
-
 cd rook
+```
+
+### Build
 
-# build rook
+```bash
+# build all rook storage providers
 make
+
+# build a single storage provider, where the IMAGES can be a subdirectory of the "images" folder:
+# "cassandra", "ceph", "cockroachdb", "edgefs", "minio", or "nfs"
+make IMAGES="cassandra" build
+
+# multiple storage providers can also be built
+make IMAGES="cassandra ceph" build
+```
+
+### Development Settings
+
+To provide consistent whitespace and other formatting in your `go` and other source files, it is recommended you apply
+the following settings in your IDE:
+- Format with the `goreturns` tool
+- Trim trailing whitespace
+
+For example, in VS Code this translates to the following settings:
+
+```json
+{
+    "editor.formatOnSave": true,
+    "go.buildOnSave": true,
+    "go.formatTool": "goreturns",
+    "files.trimTrailingWhitespace": true,
+    "files.insertFinalNewline": true,
```

Comment:
oh i like this one to insert a final new line!

Reply:
yes, thanks to @galexrt for that suggestion!

Reply:
AFAIK all of that is handled automatically by the Go extension. My `files.insertFinalNewline": true` is not set and I get the new empty line anyways.